mozci/console/commands/decision.py

```diff
@@ -19,6 +19,7 @@ class DecisionCommand(Command):
         {branch=autoland : Branch the push belongs to (e.g autoland, try, etc).}
         {--nb-pushes=15 : Do not create tasks on taskcluster, simply output actions.}
         {--dry-run : Do not create tasks on taskcluster, simply output actions.}
+        {--environment=testing : Environment to analyze (testing, production, ...)}
```

Comment:
Nit: ```suggestion
        {--environment=testing : Environment in which the analysis is running (testing, production, ...)}
```
same for the other three instances of this